com.seventhstring.Transcribe.yml

```diff
@@ -0,0 +1,61 @@
+app-id: com.seventhstring.Transcribe
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/transcribe/transcribe
+rename-desktop-file: seventhstring-transcribe.desktop
+rename-icon: seventhstring-transcribe
+finish-args:
+#TODO: figure out if its possible to only grant permissions 
+# on ~/.Transcribe!7 and xdg-music:ro, for now we use home
+# as a functional compromise
+- --share=ipc
+- --socket=x11
+- --socket=fallback-x11
+- --socket=pulseaudio
+- --filesystem=home
+modules:
+  - name: transcribe
+    buildsystem: simple
+    build-commands:
+    - bash install-linux.sh -i
+    build-options:
+      env:
+        # the install script installs into $HOME/bin and is not configurable
+        #
+        HOME: /app
+        XDG_DATA_HOME: /app/share
+    sources:
+      - type: archive
+        url: https://www.seventhstring.com/xscribe/downlo/xscsetup-9.10.0.tar.gz
+        sha256: 1eb061356c02b0c11e600519d793abb33c9af4c01d3efcd6b8c95fb5a015e58b
+    modules:
+      # the build script uses xdg-utils
+      #
+      # taken from https://github.com/flathub/com.calibre_ebook.calibre
+      - name: xdg-utils
+        buildsystem: autotools
+        cleanup: ['*']
+        sources:
+          - type: archive
+            url: https://portland.freedesktop.org/download/xdg-utils-1.1.3.tar.gz
+            sha1: 98294cf332c341b85e481d98e9ea59357fe1efc7
```

Comment:
```suggestion
```

Reply:
Why should the hash be removed? should it be sha256 instead?

Reply:
ah sorry, misread what you meant